### PR TITLE
Gordyd/new note

### PR DESCRIPTION
**Pull request by GordyD**
Added in .png icon to layout of tideline and added hover effect to the image.

screen shot 2015-08-20 at 13 19 11
screen shot 2015-08-20 at 13 20 26

### CODE REVIEW

**Comment by GordyD** [comment]
@skrugman can we do in-browser review of this please? would like to get your feedback :)

**Comment by skrugman** [comment]
Yes!

**Comment by GordyD** [comment]
Note hover text has been updated to look like this:

screen shot 2015-08-26 at 09 43 15

**Comment by skrugman** [comment]
Looks good from here but can I see it in context of the whole screen?

*Sara Krugman *
Design lead

*Tidepool *An open source, not-for-profit effort to build an open data
platform and better applications that reduce the burden of Type 1 Diabetes
and accelerate the commercialization of closed-loop systems.


*Twitter : *@skrugman
*Phone :* 617 549 1483
*Email : *sara@tidepool.org
*Web : *Tidepool.org <http://tidepool.org/>

File: js/plot/message.js [hunk 1]
```diff
@@ -129,6 +131,58 @@ module.exports = function(pool, opts) {
...
+        width: 36,
+        height: 29
+      });
+
+    newNote.on('mouseover', function() {
+      d3.select('#tidelineLabels').append('text')
+      .attr({
+        'class': 'newNoteText',
+        x: 1,
+        y: 88,
```
Comment (jebeck) [inline js/plot/message.js]: Oi, anything we can do about these magic numbers? Are they related to the size of the image asset?

File: js/plot/message.js [hunk 2]
```diff
@@ -129,6 +131,58 @@ module.exports = function(pool, opts) {
...
+      })
+      .text('New');
+      d3.select('#tidelineLabels').append('text')
+      .attr({
+        'class': 'newNoteText',
+        x: 1,
+        y: 103,
```
Comment (jebeck) [inline js/plot/message.js]: Ditto re: magic numbers.

**Comment by jebeck** [comment]
The affordance looks really different from the notes plotted in tideline on my retina display. Do we need a double-resolution asset?

**Comment by jebeck** [comment]
We decided to deploy and add in the double-res asset later, so merging.

**Pull request merged**